### PR TITLE
Add Ability to track Work Assignment ID

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8314

Matching the pattern in place, most similar to the purchase order number summary component. 

## Testing
**URL to test:** 

There are database changes, so you should test on test or locally. 

**Steps to test:**

Open a project's summary page and there is a new work assignment ID field. Update, edit, delete etc.
 
Screen Shot 2022-01-31 at 1 59 59 PM

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

**Review by amenity — APPROVED** [review]
Perfect, thanks @chiaberry!

File: moped-database/migrations/1643650785073_alter_table_public_moped_project_add_column_work_assignment_id/up.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."moped_project" ADD COLUMN "work_assignment_id" text NULL;
```
Comment (frankhereford) [inline moped-database/migrations/1643650785073_alter_table_public_moped_project_add_column_work_assignment_id/up.sql:1]: TIL there are no performance benefits of using `varchar(field_length)` in PostgreSQL. Very cool.

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ const ProjectSummaryProjectDO = ({
           <>
             <TextField
               fullWidth
-              id="moped-project-website"
+              id="moped-project-purchase-order"
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js:119]: This change looks good, but just to make sure I'm getting the big picture, this is a tag-along change in this PR right?

**Review by frankhereford — APPROVED** [review]
I've tested this locally - added, deleted, checked for persistence, etc on multiple projects. Works great - thanks Chia!

waynes-world-mike-myers

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js (hunk 2 above)
Comment (chiaberry) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js:119]: yes, sorry for forgetting to make a note. I noticed this, the TextField for purchase orders has an id for moped-project-website. 

Actually, when double checking just now, every text field has an id of "moped-project-website" on the summary page. I'll fix that in a later PR.

Comment (frankhereford) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js:119]: Awesome - thank you! That makes perfect sense. Thanks for catching those reused `id` attribute values!

Comment (johnclary) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDO.js:119]: thanks y'all!

**Comment by johnclary** [comment]
thanks @chiaberry !

**Pull request merged**